HumanCellAtlas/schema-test-data
# Determine process uuid from concatenated uuids of protocols, inputs and outputs

Related to ebi-ait/dcp-ingest-central#273

### Test plan
1. Download project directory from Terra staging area
2. Run script locally
3. Verify that the process ids in links.json file are generated from concatenated list of protocol uuids, input uuids and output uuids.
